test/__snapshots__/main.test.ts.snap

```diff
@@ -45,23 +45,45 @@ Raw error text: <code>artifacts/MockRepoOwner.MockRepoName.rawError.txt</code> i
 <li><code>git clone https://github.com/MockRepoOwner/MockRepoName --recurse-submodules</code></li>
 <li>In dir <code>MockRepoName</code>, run <code>git reset --hard 57b462387e88aa7e363af0daf867a5dc1e83a935</code></li>
 <li><details><summary>Install packages (exact steps are below, but it might be easier to follow the repo readme)</summary><ol>
-  <li>In dir <code>dirA</code>, run <code>npm --prefer-offline --no-audit --no-progress --legacy-peer-deps --ignore-scripts -q</code></li>
-  <li>In dir <code>dirB/dirC</code>, run <code>npm --prefer-offline --no-audit --no-progress --legacy-peer-deps --ignore-scripts -q</code></li>
-  <li>In dir <code>dirD/dirE/dirF</code>, run <code>npm --prefer-offline --no-audit --no-progress --legacy-peer-deps --ignore-scripts -q</code></li>
+  <li>In dir <code>dirA</code>, run <code>npm install --prefer-offline --no-audit --no-progress --legacy-peer-deps --ignore-scripts -q</code></li>
+  <li>In dir <code>dirB/dirC</code>, run <code>npm install --prefer-offline --no-audit --no-progress --legacy-peer-deps --ignore-scripts -q</code></li>
+  <li>In dir <code>dirD/dirE/dirF</code>, run <code>npm install --prefer-offline --no-audit --no-progress --legacy-peer-deps --ignore-scripts -q</code></li>
 </ol></details>
-<li>Back in the initial folder, download <code>artifacts/MockRepoOwner.MockRepoName.replay.txt</code> from the <a href="PLACEHOLDER_ARTIFACT_FOLDER">artifact folder</a></li>
+<li>Back in the initial folder, download <code>RepoResults123/MockRepoOwner.MockRepoName.replay.txt</code> from the <a href="PLACEHOLDER_ARTIFACT_FOLDER">artifact folder</a></li>
 <li><code>npm install --no-save @typescript/server-replay</code></li>
 <li><code>npx tsreplay ./MockRepoName ./MockRepoOwner.MockRepoName.replay.txt path/to/tsserver.js</code></li>
 <li><code>npx tsreplay --help</code> to learn about helpful switches for debugging, logging, etc</li>
 </ol>
+
+
+<h5>Bash script</h5>
+
+\`\`\`bash
+git clone https://github.com/MockRepoOwner/MockRepoName --recurse-submodules
+git -C "./MockRepoName" reset --hard 57b462387e88aa7e363af0daf867a5dc1e83a935
+# Install packages (exact steps are below, but it might be easier to follow the repo readme)
+npm --prefix "./dirA" install --prefer-offline --no-audit --no-progress --legacy-peer-deps --ignore-scripts -q
+npm --prefix "./dirB/dirC" install --prefer-offline --no-audit --no-progress --legacy-peer-deps --ignore-scripts -q
+npm --prefix "./dirD/dirE/dirF" install --prefer-offline --no-audit --no-progress --legacy-peer-deps --ignore-scripts -q
+downloadUrl=$(curl -s "PLACEHOLDER_GETARTIFACTS_API?artifactName=RepoResults123&api-version=7.0" | jq -r ".resource.downloadUrl")
+wget -O RepoResults123.zip "$downloadUrl"
+unzip -p RepoResults123.zip RepoResults123/MockRepoOwner.MockRepoName.replay.txt > MockRepoOwner.MockRepoName.replay.txt
+npm install --no-save @typescript/server-replay
+\`\`\`
```

Comment:
Can we just have one of these? Maybe just the latter?

I beleive the original PR had removed the instructions in favor of a download link to a shell script, which is what I was saying would be less usable, but I think this has now jumped to the other side. I think including a shell script block is totally sufficient.

Reply:
Sorry for the confusion. This PR introduced the "Bash script" to be shown and replace the "prose steps". I thought you wanted to have both of them.

For clarification, you prefer NO prose steps and ONLY the bash commands as below, correct?

![image](https://github.com/microsoft/typescript-error-deltas/assets/13305542/e5b9e222-6b4a-4c61-ac46-f54459c80e35)

Reply:
Yes, super sorry about the confusion...

Reply:
No worries! Done.